src/session/mod.rs

```diff
@@ -63,6 +63,10 @@ mod imp {
             Content::static_type();
             Self::bind_template(klass);
 
+            klass.install_action("content.go-back", None, move |widget, _, _| {
+                let self_ = Self::from_instance(widget);
+                self_.leaflet.set_visible_child(&*self_.sidebar);
```

Comment:
```suggestion
                self_.leaflet.navigate(adw::NavigationDirection::Back);
```

Just to keep it coherent with the swiping back gesture, as it uses the same function.